src/components/Form/Form.js

```diff
@@ -0,0 +1,102 @@
+import * as React from "react";
+import classNames from "classnames";
+
+import "./Form.css";
+
+import { Input } from "../Input";
+import { InputBlock } from "../InputBlock";
+
+import { Button } from "../Button";
+import { useState } from "react";
+
+export const Form = (props, { className }) => {
+  let [formState, setFormState] = useState('');
+
+  // function handleValue(event) {
+  //   setValue(event.target.value);
+  //   console.log(value)
+  // }
+  function handleChange(event) {
+    setFormState(event.target.value);
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    props.onSubmit(formState);
+
+    setFormState('')
+  }
```

Comment:
If the state moved to the  component it is not required to repeat it here. We may pass it with props together with the setState function.

You will get how to work with state in React by reading and practicing [this topic](https://reactjs.org/docs/state-and-lifecycle.html).